brainzutils/cache.py

```diff
@@ -273,19 +253,15 @@ def gen_key(key, *attributes):
 
 def _prep_dict(dictionary, namespace=None, encode=True):
     """Wrapper for _prep_key and _encode_val functions that works with dictionaries."""
```

Comment:
we should have more detailed docs here about what the format of `dictionary` is

Reply:
Makes sense, will do.